fix(srv): add info on function concurrency (#4527)
scaleway/docs-content: pages/serverless-functions/reference-content/functions-limitations.mdx

| Environment variables                               | Max size   | 1000 chars       | Environment variable        |
| Secret environment variables                        | Max number | 100              | Function + Namespace        |
| Secret environment variables                        | Max size   | 65536 bytes      | Secret environment variable |
| Time before scale to zero                           | Time       | 15 minutes       | Instance                    |
| Time before scale down                              | Time       | 30 seconds       | Instance                    |
| Max HTTP request duration\****                      | Max        | 15 minutes      | Request                     |
| Payload                                             | Max size   | 6 MiB            | Request                     |
| Logs                                                | Logs       | 30000 per minute | Project                     |

\* Lower limits may apply before account verification. [Contact our support team](https://console.scaleway.com/support) if you have any questions.

\** Total function memory is the sum of the memory allocated to all your functions at their maximum Scale.

> \*** Each instance of a function can handle a single request at a time, but a function can have several instances running in parallel.

Reviewer: ```suggestion
\*** Each instance of a function can handle a single request at a time, but a function can have several instances running at the same time.
```
